src/js/outreach/apps/error_app.js

```diff
@@ -0,0 +1,31 @@
+import { bind } from 'underscore';
+
+import RouterApp from 'js/base/routerapp';
+
+import { DialogView } from 'js/outreach/views/dialog_views';
+import { ErrorView } from 'js/outreach/views/error_views';
+
+export default RouterApp.extend({
+  eventRoutes: {
+    'error': {
+      route: 'outreach/500',
```

Comment:
When a `500` error occurs, should we just keep the user on the same URL? Like we do for the `404`?

Currently, the URL is redirected to `roundingwell.com/500` and this is shown in the UI:

![Screenshot 2023-08-17 at 3 05 05 PM](https://github.com/RoundingWell/care-ops-frontend/assets/35355575/d6ef4191-c421-4531-b36b-ff170fe587fe)

Reloading the page won't work since we redirected them to `roundingwell.com/outreach/500`. Or maybe the messaging in the UI needs to be changed.

Reply:
Okay, I'm going to do that in a different PR and shortcut story, so Sofia can track it better.